pkg/logs/logs.go

```diff
@@ -164,3 +165,15 @@ func NewContext(ctx context.Context, l logr.Logger, names ...string) context.Con
 func V(level int) klog.Verbose {
 	return klog.V(klog.Level(level))
 }
+
+type LogWithFormat struct {
+	logr.Logger
+}
+
+func WithInfof(l logr.Logger) *LogWithFormat {
+	return &LogWithFormat{l}
+}
+
+func (l *LogWithFormat) Infof(format string, a ...interface{}) {
+	l.Info(fmt.Sprintf(format, a...))
+}
```

Comment:
Maybe worth a comment or two explaining why this is here and whether in future we expect to be able to just use klog directly?

Reply:
done